CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - `starknet_estimateMessageFee` for JSON-RPC v0.3.1 to estimate message fee from L1 handler.
+- Optional CLI args `head-poll-interval-seconds` & `pending-poll-interval-seconds` to make head and pending poll intervals configurable, while preservin existing values as defaults.
```

Comment:
I think split into two items

```suggestion
- configuration for new block polling interval
- configuration for sync pending block polling interval
```

Actually, come to think of it.. do we need two different config items? I know it's what we currently have in code, but maybe we just need one? And then the option to enable pending.. wdyt?